build.gradle

```diff
@@ -154,9 +154,14 @@ subprojects {
         testRuntimeOnly "org.slf4j:slf4j-log4j12:$slf4jVersion"
     }
 
-    tasks.named('test') {
+    test {
```

Comment:
Please revert this and read the Gradle docs https://docs.gradle.org/current/userguide/task_configuration_avoidance.html